db/migrate/20220519134603_add_url_to_pointless_feedback_messages.pointless_feedback.rb

```diff
@@ -0,0 +1,6 @@
+# This migration comes from pointless_feedback (originally 20220518205500)
+class AddUrlToPointlessFeedbackMessages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :pointless_feedback_messages, :url, :string
+  end
+end
```

Comment:
did you run the generator to get this migration here?

Reply:
Yes

Reply:
Or maybe no, I ran this: `bundle exec rake pointless_feedback:install:migrations`

Reply:
Was that the correct way to do that or should I have done something else?

Reply:
yeah, that looks right. I've never seen that work for newer migrations that have been added, but cool that it does.